test_notebooks/trajectory_visualization.py

```diff
@@ -39,44 +39,25 @@ def download_trajectory_data(file):
         return (file_csv, file_exists)
 
 
-def sample_data():
+def csv_writer():
     """
-    Generates sample .csv file for unit tests.
+    Creates a sample dataset for unit tests called output.csv
     """
-
-    if os.path.exists('sample_data.csv'):
-        print(filename, 'already exists')
+    if os.path.exists('output.csv'):
+        print('output.csv', 'already exists')
 
     else:
-        c = csv.writer(open("sample_data.csv"), "wb")
-
-        # Indicate that the 1st row
-        # should be treated as column names:
-        c.put_HasColumnNames(True)
-
-        c.SetColumnName(0, "year")
-        c.SetColumnName(1, "color")
-        c.SetColumnName(2, "country")
-        c.SetColumnName(3, "food")
-
-        c.SetCell(0, 0, "2001")
-        c.SetCell(0, 1, "red")
-        c.SetCell(0, 2, "France")
-        c.SetCell(0, 3, "cheese")
 
-        c.SetCell(1, 0, "2005")
-        c.SetCell(1, 1, "blue")
-        c.SetCell(1, 2, "United States")
-        c.SetCell(1, 3, "hamburger")
-
-        #  Write the CSV to a string and display:
-        csvDoc = c.saveToString()
-        print(csvDoc)
-
-        #  Save the CSV to a file:
-        success = c.SaveFile("sample_data.csv")
-        if not success:
-            print(c.lastErrorText())
+        data = ["first_name,last_name,city".split(","),
+                "Tyrese,Hirthe,Strackeport".split(","),
+                "Jules,Dicki,Lake Nickolasville".split(","),
+                "Dedric,Medhurst,Stiedemannberg".split(",")
+                ]
+        path = "output.csv"
+        with open(path, "w") as csv_file:
+            writer = csv.writer(csv_file, delimiter=',')
+            for line in data:
+                writer.writerow(line)
 
 
 def define_xydata(dataset, sets):
```

Comment:
starting line 88:
In my opinion, the code is a little excessively spaced, hurts readability.